wildlifelicensing/apps/applications/models.py

```diff
@@ -157,6 +166,10 @@ def is_deletable(self):
         """
         return self.customer_status == 'draft' and not self.lodgement_number
 
+    @property
+    def is_licence_amendment(self):
+        return self.application_type == Application.TYPE_AMENDMENT
+
```

Comment:
I was thinking now that there's the `application_type` field that parts of the code base that make the is_licence_amendment call would now use app.application_type == 'amendment'